easybuild/easyconfigs/x/xalt/xalt-2.7.27-gpu.eb

```diff
@@ -0,0 +1,61 @@
+easyblock = 'EB_XALT'
+
+name = 'xalt'
+version = '2.7.27'
+# This does not work since XALT automatically adds xalt/<version> to
+# the install prefix and does not recognize xalt/<version>-gpu.
+#versionsuffix = '-gpu'
+
+homepage = 'https://github.com/xalt/xalt'
+description = """XALT 2 is a tool to allow a site to track user executables
+  and library usage on a cluster. When installed it can tell a site what are
+  the top executables by Node-Hours or by the number of users or the number
+  of times it is run. XALT 2 also tracks library usage as well. XALT 2 can
+  also track package use by R, MATLAB or Python. It tracks both MPI and
+  non-MPI programs."""
+
+toolchain = SYSTEM
+
+# The XALT site filter config file is site specific and must be specified
+# to successfully build XALT.  Use "--try-amend=config_py=<file>" when
+# building this EasyConfig to specify the site filter config file.
+config_py = None
+
+# Extra options
+#syshost = 'hardcode:cluster'
+#transmission = 'syslog'
+
+configopts = '--with-MySQL=no --with-trackGPU=yes'
```

Comment:
Should we have another easyconfig parameter for `--with-MySQL` in the `XALT` easyblock?